WORKING_GROUPS.md

```diff
@@ -272,4 +230,228 @@ The [Node.js Code of Conduct][] applies to this WG.
+
+* [Website](#website)
+* [Streams](#streams)
+* [Build](#build)
+* [Diagnostics](#diagnostics)
+* [i18n](#i18n)
+* [Docker](#docker)
+* [Addon API](#addon-api)
+* [Benchmarking](#benchmarking)
+* [Post-mortem](#post-mortem)
+* [Intl](#intl)
+
+
+### [Website](https://github.com/nodejs/nodejs.org)
+
+The website Working Group's purpose is to build and maintain a public
+website for the Node.js project.
+
+Responsibilities include:
+* Developing and maintaining a build and automation system for nodejs.org.
+* Ensuring the site is regularly updated with changes made to Node.js, like
+  releases and features.
+* Fostering and enabling a community of translators.
+
+### [Streams](https://github.com/nodejs/readable-stream)
+
+The Streams Working Group is dedicated to the support and improvement of the
+Streams API as used in Node.js and the npm ecosystem. We seek to create a
+composable API that solves the problem of representing multiple occurrences
+of an event over time in a humane, low-overhead fashion. Improvements to the
+API will be driven by the needs of the ecosystem; interoperability and
+backwards compatibility with other solutions and prior versions are paramount
+in importance.
+
+Responsibilities include:
+* Addressing stream issues on the Node.js issue tracker.
+* Authoring and editing stream documentation within the Node.js project.
+* Reviewing changes to stream subclasses within the Node.js project.
+* Redirecting changes to streams from the Node.js project to this project.
+* Assisting in the implementation of stream providers within Node.js.
+* Recommending versions of `readable-stream` to be included in Node.js.
+* Messaging about the future of streams to give the community advance notice of
+  changes.
+
+### [Build](https://github.com/nodejs/build)
+
+The Build Working Group's purpose is to create and maintain a distributed
+automation infrastructure.
+
+Responsibilities include:
+* Producing packages for all target platforms.
+* Running tests.
+* Running performance testing and comparisons.
+* Creating and managing build-containers.
+
+### [Diagnostics](https://github.com/nodejs/diagnostics)
+
+The Diagnostics Working Group's purpose is to surface a set of comprehensive,
+documented, and extensible diagnostic interfaces for use by Node.js tools and
+JavaScript VMs.
+
+Responsibilities include:
+* Collaborating with V8 to integrate `v8_inspector` into Node.js.
+* Collaborating with V8 to integrate `trace_event` into Node.js.
+* Collaborating with Core to refine `async_wrap` and `async_hooks`.
+* Maintaining and improving OS trace system integration (e.g. ETW, LTTNG, dtrace).
+* Documenting diagnostic capabilities and APIs in Node.js and its components.
+* Exploring opportunities and gaps, discussing feature requests, and addressing
+  conflicts in Node.js diagnostics.
+* Fostering an ecosystem of diagnostics tools for Node.js.
+
+### i18n
+
+The i18n Working Groups handle more than just translations. They
```

Comment:
Great idea. The TSC/CTC doesn't pay a whole lot of attention to those groups. And active people in those groups may be a good fit for CommComm so there can be some...uh...(buzzword alert! look away!) synergy.